mozilla_django_oidc_db/migrations/0002_migrate_to_claim_field.py

```diff
@@ -6,10 +6,14 @@
 
 import mozilla_django_oidc_db.fields
 import mozilla_django_oidc_db.models
+import mozilla_django_oidc_db.settings as oidc_settings
 
 
 def flush_cache():
-    if not (cache_name := getattr(settings, "SOLO_CACHE", "")):
+    cache_name = getattr(
+        settings, "OIDC_CACHE", oidc_settings.MOZILLA_DJANGO_OIDC_DB_CACHE
```

Comment:
I think this should try to get `MOZILLA_DJANGO_OIDC_DB_CACHE` from `django.conf.settings` first and fall back on the `oidc_settings` if it's not defined, otherwise it could differ from the configured `MOZILLA_DJANGO_OIDC_DB_CACHE` in a project that overrides it

Reply:
uh, is `CachingMixin.clear_cache` wrong then? Since that's where I grabbed the one-liner from.

I can see that `set_to_cache` indeed uses this setting, but `clear_cache` uses another one :thinking:

Reply:
Hmm you're right :thinking:. I think that `clear_cache` should indeed use `MOZILLA_DJANGO_OIDC_DB_CACHE`, since this is the setting that is used in the library (and also the one that is set in OF itself)